### DISCUSSION TITLE
Le code 'Userlike ne marche pas

### DISCUSSION THREAD

**Discussion by timrooster** [AmauriC/tarteaucitron.js#1240] [Q&A]
J’ai un problem avec le code d’Userlike.

Si j'implémente le code comme dans l'exemple de https://tarteaucitron.io/en/install/ alors cela ne fonctionne pas:

```
tarteaucitron.user.userlikekey = 'xxxx.js';
(tarteaucitron.job = tarteaucitron.job || []).push('userlike');
```

Après accepcter les cookies, le chatbot ne se charge pas. Cependant, si j'implémente un service personnalisé comme celui ci-dessous, cela fonctionne.

``` tarteaucitron.services.userlikenew = {
"key": "userlikenew",
"type": "support",
"name": "Userlike",
"uri": "https://www.userlike.com/en/terms#privacy-policy",
"needConsent": true,
"cookies": ['uslk_s', 'uslk_e'],
"js": function () {
    "use strict";
    tarteaucitron.addScript('//userlike-cdn-widgets.s3-eu-west-1.amazonaws.com/xxxx.js');
}

(tarteaucitron.job = tarteaucitron.job || []).push('userlikenew‘); 
```

C'est juste une copie du code du fichier tarteaucitron.services.js avec userlike renommé userlikenew. 
Pourquoi celui fonctionne mais le code original ne fonctionne pas?

**Comment by AmauriC** [comment]
Pourtant le rendu est le rendu au final est le même.
Possible d'avoir une url qui reproduit le problème?

**Comment by timrooster** [comment]
Nous avons pu résoudre le problème

Il y a une erreur dans les instructions sur https://tarteaucitron.io/en/install/ pour Userlike.

Dans les instructions il dit:
   tarteaucitron.user.userlikekey = 'userlikekey';

Mais dans le fichier tarteaucitron.services.js le nom de la variable est tarteaucitron.user.userlikeKey. Avec un K capitale.

Alors, "tarteaucitron.user.userlikeKey = 'xxx';" fonctionne.

**Comment by AmauriC** [comment]
Merci! J'ai corrigé la doc.
